integrationtests/tests/fixtures/mod.rs

```diff
@@ -1206,6 +1207,15 @@ impl FederationTest {
             let mut modules = BTreeMap::new();
             let env_vars = FedimintConsensus::get_env_vars_map();
 
+            fedimint_core::db::apply_migrations(
+                &db,
+                "Consensus".to_string(),
```

Comment:
Should it be "consensus" or some kind of "fedimint-core" / "global" or something like that? It's not just consensus data we might migrate. Anything that generic fedimint server is storing might be subject to migration.

Reply:
Yeah "Global" is probably better